program_v2/tests/suite-resolve/resolve.ts

```diff
@@ -179,22 +194,283 @@ describe('sol-did resolve and migrate operations', () => {
     );
   });
 
-  it('can successfully update the state of a DID', async () => {
+  it('can update the verificationMethods of a Did', async () => {
+    const existing = await service.getDidAccount();
+    expect(existing).to.not.be.equal(null);
+    let vms = [
+      {
+        fragment: newSolKeyAlias,
+        keyData: newSolKey.publicKey.toBytes(),
+        methodType: VerificationMethodType.Ed25519VerificationKey2018,
+        flags: VerificationMethodFlags.CapabilityInvocation,
+      },
+      {
+        fragment: 'new-key',
+        keyData: newSolKey.publicKey.toBytes(),
+        methodType: VerificationMethodType.Ed25519VerificationKey2018,
+        flags: VerificationMethodFlags.CapabilityDelegation,
+      },
+    ];
+    const updated = await service
+      .update({
+        controllerDIDs: [],
+        services: [],
+        verificationMethods: vms,
+      })
+      .withSolWallet(authority)
+      .rpc();
+    let updated_account = await service.getDidAccount();
+    expect(updated_account?.services.length).to.equal(0);
+    expect(updated_account?.verificationMethods.length).to.equal(2);
+    expect(updated_account?.nativeControllers.length).to.equal(0);
+    expect(updated_account?.otherControllers.length).to.equal(0);
+  });
+
+  it('cannot update the verificationMethods of a Did if there are replications', async () => {
+    const existing = await service.getDidAccount();
+    expect(existing).to.not.be.equal(null);
+    let vms = [
```

Comment:
can you move the generation of payloads outside of the testfile? e.g. into `test/utils/utils.ts`